### ISSUE TITLE
Possible reframing of this workshop or second workshop

### ISSUE THREAD

**Issue by pattyf** [dlab-berkeley/sql-for-r-users#16]
I like the idea of an SQL workshop because it is so important in the professional world. We had a lot of folks register for the two iterations of this workshop, underscoring interest in this topic. A Python version of this workshop would be great too.But we have so many workshops and not enough time. I wonder if we could create a generic SQL workshop that uses the dbeaver app (https://dbeaver.io) to teach SQL queries and then spend the last half hour of the workshop demonstrating how to read the results of a SQL query into an R dataframe or a Pandas dataframe.

**Comment by aculich** [comment]
@pattyf yes, I very much agree that we need to take another look at our SQL and database offerings and think about what to offer in this space. We get a number of consulting requests that would benefit from not only some best practices recommendations at the ready, but also a couple of D-Lab workshops that people can take. I'm interested in helping to develop and/or teach some of these, too.
